### ISSUE TITLE
Indent: warn for multiline function calls

### ISSUE THREAD

**Issue by zpao** [eslint/eslint#4197]
This does not fail but I believe it should. The example is a bit contrived, but is a pattern that is used when calling a function with a number of arguments and it doesn't fit inside a line length rule. I just checked in master (added a test case) so I can confirm it's not fixed by any recent releases.

```js
/*eslint indent: [2, 2]*/
console.log(
    'foo'
)
```

I can see where this might be a bit of a grey area but I think it makes a lot sense for the `indent` rule to catch that this was indented 4 characters instead of 2. Thanks for the consideration.

cc @gyandeeps since you've been making lots of changes to the `indent` rule (thanks by the way!)

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
I think this falls along the lines of the following open issue: #1801

**Comment by zpao** [comment]
Ah, I did a quick search but didn't catch that one. I think this is essentially the same.

**Comment by gyandeeps** [comment]
closing in favor of #1801
